admin/import/class-import-settings.php

```diff
@@ -105,7 +105,12 @@ private function determine_path() {
 		$this->path = $this->upload_dir['basedir'] . DIRECTORY_SEPARATOR . 'wpseo-import' . DIRECTORY_SEPARATOR;
 
 		if ( ! isset( $GLOBALS['wp_filesystem'] ) || ! is_object( $GLOBALS['wp_filesystem'] ) ) {
-			WP_Filesystem();
+			$url = wp_nonce_url(
+				self_admin_url('admin.php?page=wpseo_tools&tool=import-export'),
+				'wpseo-import'
+			);
+			$creds = request_filesystem_credentials( esc_url_raw( $url ) );
```

Comment:
I am not a big fan of abbreviations and in our codebase we try to avoid them as far as possible. Can you change `$creds` into `$credentials`